src/style/markdown.css

```diff
@@ -55,10 +51,6 @@
 		font-weight: bold;
 		cursor: text;
 		position: relative;
-
-		&:hover a.anchor {
-			text-decoration: none;
-		}
```

Comment:
Unused, `.anchor` goes back to the repo creation but we no longer have anything that uses it